server/StudySharp.API/Requests/TheoryBlocks/UpdateTheoryBlockByIdRequest.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace StudySharp.API.Requests.TheoryBlocks
+{
+    public class UpdateTheoryBlockByIdRequest
```

Comment:
field Name is missing

Reply:
TheoryBlockId and CourseId should be grabbed from Route, Name should be grabbed from Body